### ISSUE TITLE
Import statements don't seem to be handled properly.

### ISSUE THREAD

**Issue by Sean1708** [tonyhffong/Lint.jl#85]
Personally, I don't think this should give an error:

```julia
julia> using Lint

julia> lintstr("""
       import Gadfly
       Gadfly.plot(x=1:10, y=1:10)
       """)
UndefVarError(:Gadfly)
$(Expr(:import, :Gadfly))
1-element Array{LintMessage,1}:
 none:2 [               ] ERROR  Use of undeclared symbol Gadfly
```

Comparing it to a `using` equivalent:

```julia
julia> using Lint

julia> lintstr("""
       using Gadfly
       Gadfly.plot(x=1:10, y=1:10)
       """)
0-element Array{LintMessage,1}
```

It seems to me that `using` gets `eval`ed (or something similar) whereas `import` doesn't, which I think would explain this behaviour:

```julia
julia> using Lint

julia> import Gadfly

julia> lintstr("""
       import Gadfly
       Gadfly.plot(x=1:10, y=1:10)
       """)
0-element Array{LintMessage,1}
```

I get the same behaviour regardless of the package and I get equivalent behaviour when using `lintfile` and `lintpkg`.

**Comment by tonyhffong** [comment]
Could you check to see if the latest master behaves properly?

**Comment by Sean1708** [comment]
That's working perfectly for me, thanks.
